.github/workflows/resync-apply.yaml

```diff
@@ -1,14 +1,14 @@
 name: Resync Apply "v1"
 
-on:
-  push:
-    branches:
-      - main
-    paths:
-      - "cognite/powerops/resync/v1/**.py"
-      - "cognite/powerops/cli.py"
-      - "tests/data/demo/v1/**.yaml"
-      - "tests/data/demo/v1/**.yml"
+on: workflow_dispatch
```

Comment:
**question:** not sure if this is the correct format or not. I've only ever seen it defined as I've suggested (could be that yours ends up being parsed the same, but idk)

```suggestion
on:
    workflow_dispatch:
```